pydicom/filereader.py

```diff
@@ -1008,22 +1021,41 @@ def read_deferred_data_element(fileobj_type, filename_or_obj, timestamp,
           if is_filename else filename_or_obj)
     is_implicit_VR = raw_data_elem.is_implicit_VR
     is_little_endian = raw_data_elem.is_little_endian
+
+    if config._memmap_size and raw_data_elem.length > config._memmap_size:
+        # Reading value via memmap and return numpy.memmap array dtype=uint8
+        try:
+            import numpy
+        except ImportError:
+            raise ImportError(
+                "Numpy required for memmap'd values, but numpy is missing"
+            )
+        mode = 'r' if config._memmap_read_only else 'r+'
```

Comment:
I was already asking here why not use the Python module, and why not map the whole file - but now noticed that @scaramallion had asked exactlt the same :) Makes sense to me, too.